packages/react-components/src/templates/UserProfileHeader.tsx

```diff
@@ -28,6 +27,7 @@ const containerStyles = css({
     ".             edit-personal-info" ${24 / perRem}em
     "personal-info personal-info     " auto
     "contact       edit-contact-info " auto
```

Comment:
Yeah I'm afraid I can't approve this with the duplicate `<SocialIcons>` :sweat_smile: it would be the very first time we duplicate DOM elements to do a `display: none` switch at a breakpoint.
It would seem to me that in order to have the one `SocialIcons` that is immediately inside the grid always placed correctly, we could extend the desktop grid's `1fr` column to two columns: `auto 1fr` or `max-content 1fr` or so, `contact social`. Then we have only one `SocialIcons` (the "non-inline" one) that we can place on both mobile and desktop.
We can pair on this tmrw